src/main/java/cn/edu/tsinghua/tsfile/timeseries/read/FileReader.java

```diff
@@ -299,4 +300,32 @@ public TsFileMetaData getFileMetaData() {
         return this.fileMetaData;
     }
 
+    public List<RowGroupMetaData> getSortedRowGroupMetaDataList() throws IOException{
```

Comment:
This method is lack of comments.
Some comments should be emphasized: this method is only used for hadoop.

Reply:
This method is never used?